CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
 # CHANGELOG
 
+## 1.29.0 (2022-09-16)
+
+MLflow 1.29.0 includes several major features and improvements
+
+Features:
+
+- [Tracking] Add support for load_text, load_image, load_dict to the fluent API (#6475, @subramaniam02)
+- [Tracking] Add attributes `creation_time` and `last_update_time` to the Experiment class (#6756, @subramaniam02)
+- [Tracking] Add a default generated run name to started runs if not supplied by user (#6736, @BenWilson2)
+- [Tracking] Add `pos_label` to `mlflow.evaluate` to define the positive condition value in binary classification evaluations (#6696, @harupy)
+- [Tracking] Update RunInfo, CreateRun and UpdateRun classes to accept `run_name` as a named attribute (#6769, @apurva-koti)
+- [Build / Docker] Add MLflow Tracking Server DockerFile and Docker compose files (#6731, @oojo12)
+- [Build / Docker] [CLI] Add new CLI command for generating Dockerfiles for models (#6591, @anuarkaliyev23)
+- [Build] Add Docker development scripts (#6465, @hubertzub-db)
```

Comment:
Can we move this into "Small bug fixes and documentation updates" since it's a feature for platform contributors and not a platform feature for users?